packages/bcer-api/app/src/location/location.service.ts

```diff
@@ -381,12 +381,12 @@ export class LocationService {
    * @param locationIds 
    * @returns 
    */
-  async closeLocation(locationIds: string[]): Promise<UpdateResult> {
+  async closeLocation(locationIds: string[], closedAt: number = moment().unix()): Promise<UpdateResult> {
     const result = await this.locationRepository.update(
       { id: In(locationIds) },
       { 
         status: LocationStatus.Closed,
-        closedAt: moment().toDate(),
+        closedAt: moment.unix(closedAt).toDate(),
```

Comment:
I'm wondering if we maybe should add in a separate timestamp for this? So we keep the timestamp the action happened AND the date that was selected by the user. Thoughts?

Reply:
![image](https://user-images.githubusercontent.com/46575401/138358295-5e360785-2bdd-4d5d-92e1-faa8e6fecac2.png)
Fixed, please check again.